app/scripts/services/validateRegistrant.js

```diff
@@ -1,23 +1,23 @@
 'use strict';
 
 angular.module('confRegistrationWebApp')
-  .service('validateRegistrant', function validateRegistrant() {
+  .service('validateRegistrant', function validateRegistrant(util) {
 
-    var blockVisibleRuleCheck = function(block, registrant){
+    var blockVisibleRuleCheck = function (block, registrant) {
       var returnValue = true;
       var answers = registrant.answers;
-      angular.forEach(block.rules, function(rule){
-        var answer = _.find(answers, {blockId: rule.parentBlockId});
-        if(angular.isUndefined(answer) || answer.value === ''){
+      angular.forEach(block.rules, function (rule) {
+        var answer = _.find(answers, { blockId: rule.parentBlockId });
+        if (angular.isUndefined(answer) || answer.value === '') {
           returnValue = false;
-        }else{
-          if(rule.operator === '=' && answer.value !== rule.value) {
+        } else {
+          if (rule.operator === '=' && answer.value !== rule.value) {
             returnValue = false;
-          }else if(rule.operator === '!=' && answer.value === rule.value){
+          } else if (rule.operator === '!=' && answer.value === rule.value) {
             returnValue = false;
-          }else if(rule.operator === '>' && answer.value <= rule.value){
+          } else if (rule.operator === '>' && answer.value <= rule.value) {
             returnValue = false;
-          }else if(rule.operator === '<' && answer.value >= rule.value){
```

Comment:
What was wrong with Lodash's `_.isNumber` function?

Also I think you need to check the range here. This is used for validating the whole registration when the inputs aren't on the page and is used by the review registration page. I tried once to consolidate the validation Angular does on the inputs and this file but it got very complex. Ideally the app would be rearchitected to allow for the same validation config everywhere...